src/Common/Config/ConfigProcessor.cpp

```diff
@@ -19,10 +20,12 @@
 #include <common/errnoToString.h>
 #include <IO/WriteBufferFromString.h>
 #include <IO/Operators.h>
-
+#include <yaml-cpp/yaml.h>
 
 #define PREPROCESSED_SUFFIX "-preprocessed"
 
+#include <Poco/DOM/DOMWriter.h>
+#include <Poco/XML/XMLWriter.h>
```

Comment:
What is it for?

Reply:
Back then, I used XMLWriter to produce logs for yaml->xml conversion. Now we don't need them.